internal/scaler/awsec2asg/aws_ec2_asg.go

```diff
@@ -82,7 +84,9 @@ func (s *Scaler) Type() scaler.Type {
 }
 
 func (s *Scaler) Register(ctx context.Context) error {
-	sess, err := session.NewSession()
+	sess, err := session.NewSession(&aws.Config{
+		Region: aws.String(AWS_REGION),
```

Comment:
Please check for `s.Region != nil` and set the Region value here if it is present.

Reply:
setting a default value might help us if the user forgets to pass such argument in the toml file. What do you say @scriptnull

Reply:
That may lead to confusion because we can't predict a good default AWS location that works for everyone. If we set an opinionated default location, then there is a chance that we silently end up creating EC2 autoscaling groups (and VMs) in that region (which the user may not be aware of until the surprise AWS bill).

So it is better to prefer getting the region as input from the user upfront instead of us trying to suggest a default.

Reply:
If the user doesn't set it, let's handle it as an error.